### PR TITLE
Implementing Dynamic Todo List App and also used local storage for storing and fetching the data

### PR DESCRIPTION
**Pull request by sahnavin123**
# Description:
This pull request adds the functionality of creating dynamic todo items, separate application logic from the DOM-related operations, and incorporate persistence using the Web Storage API.

## Features:
- Create ToDo: Users can create new todo items with a title, description, due date, priority, and optional notes.
- Form Validation: The code includes validation checks for required fields (title, description, and due date).
- Display ToDo List: The website displays the list of todos in the UI, showing the title, due date, priority, and notes (if available).
- Delete ToDo: Users can delete individual todo items from the list.
- Form UI Interaction: The page allows to reset the form, and submit a new todo.

## page screenshot:
| image  |  image |
|:-:|:-:|

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+</head>
+<body>
+    <h1>Your To-Do App</h1>
+    <div class="content">
+        <div class="form-container">
+            <button class="btn" id="add-todo-button">Add New ToDo</button>
+            <div id="add-todo-form" style="display:none">
+                <form id="add-todo">
+                    <label for="Title">Title:</label>
+                    <input type="text" id="Title" name="Title" placeholder="Title" required>
```
Comment (devanuj97) [inline dist/index.html]: id name should not start with capital latter, please change from `Title` => `title`

also update on other place as well

Comment (devanuj97) [inline dist/index.html]: also 
```html
<input type="text" id="Description" name="Description" placeholder="Description" required>
```
to 
```html
<input type="text" id="Description" name="Description" placeholder="Description" required />
```

File: dist/index.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+                    <label for="Description">Description:</label>
+                    <input type="text" id="Description" name="Description" placeholder="Description" required>
```
Comment (devanuj97) [inline dist/index.html]: this field should be the textarea
syntax --
```html
<textarea>....</textarea>
```

File: src/createToDo.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    alert("please enter description less than 50 character");
+    return;
+  }
+
+  if (parseISO(dueDate) < startOfToday()) {
+    alert(
+      "You have entered a date that has already passed!  Please enter a date greater than or equal to today."
+    );
+    return;
+  }
```
Comment (devanuj97) [inline src/createToDo.js]: such error popping is strongly not acceptable method show the error in proper place specifically to this show error message below the form field

**Review by devanuj97 — CHANGES_REQUESTED** [review]

**Comment by devanuj97** [comment]
Screenshot 2023-06-09 at 2 51 53 PM

Align the Add TODO button container and list todo container

**Review by devanuj97 — APPROVED** [review]

File: src/createToDo.js (hunk 3 above)
Comment (sahnavin123) [reply src/createToDo.js]: did the changes

**Pull request merged**